firestore/firestore_snippets/collection_group_query.go

```diff
@@ -51,3 +51,66 @@ func collectionGroupQuery(w io.Writer, projectID string) error {
 }
 
 // [END firestore_query_collection_group_filter_eq]
+
+func partitionQuery(ctx context.Context, client *firestore.Client) error {
+	// [START firestore_partition_query]
+	cities := client.CollectionGroup("cities")
+
+	// Get a partioned query for the cities collection group, with a maximum
+	// partition count of 10
+	partitionedQueries, err := cities.GetPartitionedQueries(ctx, 10)
+	if err != nil {
+		return err
```

Comment:
Might want to use `fmt.Errorf` to add more context to the error when returning it. See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors.